org.adempiere.base/src/org/compiere/model/MOrder.java

```diff
@@ -3000,4 +3003,35 @@ public int getDocTypeID()
 		return getC_DocType_ID() > 0 ? getC_DocType_ID() : getC_DocTypeTarget_ID();
 	}
 
+	/**
+	 * 
+	 * @return prepayment amount for order
+	 */
+	public BigDecimal getPaymentAmt()
+	{
+		BigDecimal retValue = null;
+		String sql = "SELECT SUM(currencyConvert(p.PayAmt+p.DiscountAmt+p.WriteOffAmt, "
```

Comment:
Hi @hengsin - I have not tested yet, but maybe here we must use the new currencyconvertpayment function to make it compatible with IDEMPIERE-4083

Reply:
ok, changed.